### ISSUE TITLE
TANDA support

### ISSUE THREAD

**Issue by yhifny** [ThilinaRajapakse/simpletransformers#715]
Hi,

Do u have any plan to support TANDA (https://github.com/alexa/wqa_tanda). 

Thanks,
Yasser

**Comment by ThilinaRajapakse** [comment]
I haven't read the paper yet, so I might be missing something. Is there something special about the training procedure? It looks like it's just training the model on a big dataset and then on a smaller one (at first glance).

**Comment by yhifny** [comment]
The models are ready but it takes an input consists of [query, context] returns a softmax over two classes. I am not sure how to use it with your library.
